tests/testthat/test-textstat_collocations.R

```diff
@@ -263,71 +261,32 @@ test_that("deprecated collocations function works", {
         textstat_collocations(txts, size = 2, min_count = 2)
     )
     expect_warning(
-        collocations(txts, size = 3, min_count = 2),
+        collocations(txts, size = 2, min_count = 2),
         "'collocations' is deprecated"
     )
     expect_warning(
-        sequences(txts, size = 3, min_count = 2),
+        sequences(txts, size = 2, min_count = 2),
         "'sequences' is deprecated"
     )
 })
 
-test_that("tokens_segment_by_punctuation works as expected", {
```

Comment:
Can we verify that the new code produce behaviour matching the older function (tokens_segment_by_punctuation()`)?